features/list-elements.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
-name: <li>
-description: The `<li>` element TODO.
+name: <ol>, <ul>, and <li>.
```

Comment:
(this comment is here only because github doesn't let me comment on lines you didn't change)

How about using:

```
compute_from:
  - html.elements.ol
  - html.elements.ul
  - html.elements.li
```

This way we'd ignore the `reversed` attribute which makes the feature start at Chrome 18, Firefox 18, and Safari 6, but which really isn't needed in 99.9% of the cases.